apps/server/src/shared/repo/dashboard/dashboard.model.mapper.spec.ts

```diff
@@ -81,6 +82,7 @@ describe('dashboard model mapper', () => {
 						),
 					},
 				],
+				userId: new ObjectId().toString(),
 			});
 
 			const mapped = await mapper.mapDashboardToModel(dashboard);
```

Comment:
Should we check the mapped userId?

Reply:
not quite sure what you mean here

Reply:
ah, now I get it... yes we should!

Reply:
So, the test is in "dashboard.uc.spec.ts". Does this mean that is the right place for the userId to be tested? (And not "dashboard.model.mapper.ts" which seems more direct ?)

Reply:
https://github.com/hpi-schul-cloud/schulcloud-server/pull/3185/files#diff-262a3da1773a40b63e06c321ce5a90a0c76c05ce7e57d380b70c9eef4a1c87b3R55

Reply:
right there :)

Reply:
Aha, now I get it. :D